source/cl_input.c

```diff
@@ -298,10 +332,13 @@ void CL_AdjustAngles (void)
 #ifdef __PSP__
 		cl.viewangles[YAW] -= speed*cl_yawspeed.value*CL_KeyState (&in_right) * in_sensitivity.value;
 		cl.viewangles[YAW] += speed*cl_yawspeed.value*CL_KeyState (&in_left) * in_sensitivity.value;
-#else
+#elif _3DS
 		cl.viewangles[YAW] -= speed*cl_yawspeed.value*CL_KeyState (&in_right) * sensitivity.value;
 		cl.viewangles[YAW] += speed*cl_yawspeed.value*CL_KeyState (&in_left) * sensitivity.value;
-#endif // __PSP__
+#else 
+		cl.viewangles[YAW] -= speed*cl_yawspeed.value*CL_KeyState (&in_right);
```

Comment:
Instead of doing this where the this is 90% duplicate code, can you define a var that represents sensitivity on other platforms, and 1 on wii, and multiply by that var?